src/helpers/constants/socialLinkItems.ts

```diff
@@ -5,20 +5,36 @@ export const socialLinkItems = [
     text: 'Instagram',
     iconGlyph: 'instagram-logo' as keyof typeof icons,
     href: 'https://www.instagram.com/onefallma/',
+    position: 'footer',
```

Comment:
@Yura33-dev, I think I found a good explanatory name for this key: `fullListOnly`.

While Instagram, Facebook, Twitter, and TikTok are present in any social link menu (e.g., sidebar, overlay, and footer), Bandcamp and YouTube are not. Following that logic, Bandcamp and YouTube will have `fullListOnly` assigned to `true`, while the rest will be `false`.

Below how we may use it in the footer. Note that we need to explicitly destructure out `fullListOnly` and ensure it's only used for filtering, not passed to the MenuItem component. Otherwise, React will pass it to a DOM element and throw an error since it's not recognized as a valid boolean attribute.

```jsx
// Footer.tsx

<Menu type='footer-social-links'>
  {socialLinkItems
    .filter(({ fullListOnly }) => !fullListOnly)
    .map(({ fullListOnly, ...item }) => (
      <Menu.Item
        key={item.text}
        {...item}
        rel='noopener noreferrer'
        target='_blank'
      />
    ))}
</Menu>
```

We'll also need to destructure `fullListOnly` out everywhere in the project where we're mapping through `socialLinkItems`.

Please let me know if you like it better this way ;)

Reply:
Absolutely, this approach is elegant and really passed for this case)